src/Bitmovin/api/factories/muxing/MuxingFactory.php

```diff
@@ -129,28 +211,33 @@ public static function createMuxingForEncoding(JobContainer $jobContainer, Encod
             {
                 $smoothStreamingOutputFormat = $format;
             }
+            if ($format instanceof ProgressiveMp4OutputFormat)
+            {
+                $progressiveMp4OutputFormats[] = $format;
```

Comment:
Instead of introducing a new variable here, you could do this check in static::createMP4MuxingsForEncoding